python/paddle/distributed/auto_parallel/constants.py

```diff
@@ -114,6 +114,7 @@ def set_field_default_config(category, field, default_value):
 set_field_default_config(PIPELINE, "accumulate_steps", 1)
 set_field_default_config(PIPELINE, "generation_batch_size", 1)
 set_field_default_config(PIPELINE, "enable_send_recv_overlap", False)
+set_field_default_config(PIPELINE, "auto_parallel_profiler", False)
```

Comment:
```suggestion
set_field_default_config(PIPELINE, "schedule_profiler", False)
```
`auto_parallel`是`pipeline`的上位词，不应该在`pipeline`的下一级开关里出现。

Reply:
done